packages/@dcl/inspector/src/lib/babylon/decentraland/sdkComponents/gltf-container.ts

```diff
@@ -77,23 +77,30 @@ export function loadGltf(entity: EcsEntity, value: string) {
     sceneContext = entity.context
   }
 
-  const root = entity.getRoot()
+  let root = entity.getRoot()
+  let attempts = 0
 
-  const shouldHide = root === PLAYER || root === CAMERA || root === null
+  while (root === null && attempts++ < 10) {
```

Comment:
there could be a scenario where there are GLTFs and there a no Nodes (like if the user deliverately edits the `main.composite` and removes the Nodes component data), and in that case this would loop forever.

To avoid that, we kill the loop after 1s. What would happen if at such point the Nodes component is not loaded, the root will be `null`, and if the models were nested under the Camera or Player entities, they would NOT be hidden. It's a very specific edge case anyway.